### DISCUSSION TITLE
How to get the error details when process.exit(1)

### DISCUSSION THREAD

**Discussion by yisheng** [directus/directus#16439] [Archive]
For example:
```javascript
if ((await isInstalled()) === false) {
    logger.error(`Database doesn't have Directus tables installed.`);
    process.exit(1);
}
```
If directus was not installed, I only got `ELIFECYCLE  Command failed with exit code 1.`, but no error details.

Thanks.

**Comment by br41nslug** [comment]
Progress is tracked on this ticket https://github.com/directus/directus/issues/16284
